chall03/pcuadrad.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import sys
+
+def error():
```

Comment:
I think "usage" would have been a better name than "error" for this function.

Reply:
For me that name makes more sense because it is like an exception to the normal flow of the program, but it is worth thanks for the note.

Reply:
It definitely was a rather subjective observation from me.